src/AppEntry.js

```diff
@@ -7,7 +7,7 @@ import { getBaseName } from '@redhat-cloud-services/frontend-components-utilitie
 
 const InsightsRbac = () => (
   <Provider store={store}>
-    <Router basename={getBaseName(location.pathname)}>
+    <Router basename={getBaseName(location.pathname, 2).includes('rbac') ? '/settings/rbac' : 'settings'}>
```

Comment:
I think the `settings` basename should also include `/`.

(Not 100% sure on this one) You also have to take beta into consideration. I think the `getBaseName` should include the `beta` partial if its necessary.

Reply:
Yep, I wanted to make it more explicit, but `getBaseName` should handle these cases, so it will be better to use it.